.github/workflows/build_cmake.yml

```diff
@@ -196,3 +196,12 @@ jobs:
           LICENSE
           ThirdPartyNotices.txt
         retention-days: 5
+
+    - name: Upload Windows Server Debug Symbols Artifact
+      uses: actions/upload-artifact@v2
+      if: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/releases')) && (runner.os == 'Windows') }}
+      with:
+        name: ni-grpc-device-server-windows-x64-debug-symbols
+        path: |
+          build/Debug/ni_grpc_device_server.pdb
```

Comment:
This looks like symbols for a Debug build, not a RelWithDebInfo build. I would expect the RelWithDebInfo symbols to be in build/Release or build/RelWithDebInfo.

Reply:
If the files are in build/RelWithDebInfo, don't you also need to update the other steps to use that directory?

```
    - name: Upload Windows Server Binaries Artifact
      uses: actions/upload-artifact@v2
      if: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/releases')) && (runner.os == 'Windows') }}
      with:
        name: ni-grpc-device-server-windows-x64
        path: |
          build/Release/ni_grpc_device_server.exe
          build/Release/server_config.json
          build/Release/server_capabilities.json
        retention-days: 5
```